src/pages/Organization/OrganizationFacilities.tsx

```diff
@@ -35,7 +36,7 @@ export default function OrganizationFacilities({
   const { qParams, Pagination, advancedFilter, resultsPerPage, updateQuery } =
     useFilters({ limit: 15, cacheBlacklist: ["name"] });
 
-  const { data: facilities, isLoading } = useQuery({
+  const { data: facilities, isFetching } = useQuery({
```

Comment:
Not a major deal since behavior will work for page 2 onwards, but change the offset to

 `((qParams.page ?? 1) - 1) * resultsPerPage`

in facilities, patients and users components here (qParams.page is undefined for page 1).